zxcvbn-test/ZxcvbnTestMesserliExtended.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.IO.Compression;
+
+namespace zxcvbn_test
+{
+    [TestClass]
+    public class ZxcvbnTest_MesserliExtended
+    {
+        private const string CategoryUnitTest = "UnitTest";
+
+        [TestMethod]
+        [TestCategory(CategoryUnitTest)]
+        [Priority(1)]
+        public void ZxcvbnEmptyPasswordMetric()
+        {
+            var passwordChecker = new Zxcvbn.Zxcvbn();
+
+            var result = passwordChecker.EvaluatePassword("");
+
+            Assert.AreEqual(0.0, result.Entropy);
+        }
+
+        [TestMethod]
+        [TestCategory(CategoryUnitTest)]
+        [Priority(1)]
+        public void ZxcvbnPasswordMetric()
+        {
+            var passwordChecker = new Zxcvbn.Zxcvbn();
+
+            var a = passwordChecker.EvaluatePassword("a");
+            var aaaa = passwordChecker.EvaluatePassword("aaaa");
+            var abab = passwordChecker.EvaluatePassword("abab");
+            var ABab = passwordChecker.EvaluatePassword("ABab");
+
+            Assert.IsTrue(aaaa.Entropy > a.Entropy);
+            Assert.IsTrue(abab.Entropy > aaaa.Entropy);
+            Assert.IsTrue(ABab.Entropy > abab.Entropy);
+        }
+
+        [TestMethod]
+        [TestCategory(CategoryUnitTest)]
+        [Priority(1)]
+        public void DictionaryMatcherTest()
+        {
+            // We test if the dictionary matcher loads the messerli dictionary
```

Comment:
This should be the test name instead of a comment.

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/0e2ffddf6e997dae943ad50fe63e892d0ed5d24f

Reply:
name `LoadsMesserliDictionary()`

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/e6cb111a3cc6f41eb627b156c515c59ab810ee9e